reg_tests/chgres_cube/13km.conus.rap-smoke.grib2.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+#-----------------------------------------------------------------------------
+# Invoke chgres to create 13-km CONUS coldstart files using RAP GRIB2 data
```

Comment:
I would add something to the description about this testing the 'smoke' option. What it is and when it is used?

Reply:
Resolved at a4de98b.